tests/tests/swfs/avm2/escape/Test.as

```diff
@@ -4,38 +4,44 @@
 	}
 }
 
-trace("// escape()");
-trace(escape());
-trace("");
-
-trace("// escape(undefined)");
-trace(escape(undefined));
-trace("");
-
-trace("// typeof(escape(undefined))");
-trace(typeof(escape(undefined)));
-trace("");
-
-trace("// escape(null)");
-trace(escape(null));
-trace("");
-
-var input = "test";
-trace("// escape(\"" + input + "\")");
-trace(escape(input));
-trace("");
-
-var input = "!\"£$%^&*()1234567890qwertyuiop[]asdfghjkl;'#\zxcvbnm,./QWERTYUIOP{}ASDFGHJKL:@~|ZXCVBNM<>?\u0010";
-trace("// escape(\"" + input + "\")");
-trace(escape(input));
-trace("");
-
-var input = "\x05";
-trace("// escape(\"\\x05\")");
-trace(escape(input));
-trace("");
-
-var input = "😭";
-trace("// escape(\"" + input + "\")");
-trace(escape(input));
-trace("");
+import flash.utils.getDefinitionByName;
+
+var fns = ["escape", "encodeURI", "encodeURIComponent"];
+for each (var fnName in fns) {
+	var fn = getDefinitionByName(fnName);
+	trace("// " + fnName + "()");
+	trace(fn());
+	trace("");
+
+	trace("// " + fnName + "(undefined)");
+	trace(fn(undefined));
+	trace("");
+
+	trace("// typeof(" + fnName + "(undefined))");
+	trace(typeof(fn(undefined)));
+	trace("");
+
+	trace("// " + fnName + "(null)");
+	trace(fn(null));
+	trace("");
+
+	var input = "test";
+	trace("// " + fnName + "(\"" + input + "\")");
+	trace(fn(input));
+	trace("");
+
+	var input = "!\"£$%^&*()1234567890qwertyuiop[]asdfghjkl;'#\zxcvbnm,./QWERTYUIOP{}ASDFGHJKL:@~|ZXCVBNM<>?\u0010";
```

Comment:
`;'#\zxcvbnm`
Shouldn't this be `;'#\\zxcvbnm`? This was present in the original test for `escape`

Reply:
I copied this from line 28 in the original diff - there's only a single backslash there.